src/root/NetworkAgnostic.ts

```diff
@@ -23,20 +23,34 @@ const metaTransactionType = [
 ]
 
 export default class NetworkAgnostic extends ContractsBase {
-  web3Client: Web3Client
+  public biconomy: Biconomy
+  public networkAgnostic: Web3
```

Comment:
The class is named `NetworkAgnostic` and one of its state variables is named `networkAgnostic`. This might create confusion. 
According to standard conventions `networkAgnostic` should be the name of the object which is an instance of `NetworkAgnostic` class.